pkg/distro/rhel9/images.go

```diff
@@ -698,3 +708,24 @@ func initialSetupKickstart() *fsnode.File {
 	}
 	return file
 }
+
+// Creates a unit file that will create the the mountpoints if it doesnot exits.
+// This ensures that the filesystem created using image builder continues to work after any upgrade.
+func createFilesystemMounpointService(customizations *blueprint.Customizations) *fsnode.File {
```

Comment:
I think somewhere has been mentioned to create an osbuild stage just for this no?

Reply:
We can get this out for as fix and having a osbuild stage is more more like a permanent fix to a temporary problem, 
I would prefer more of a template-able service that picks ups the mountpoints and create them on the go.
cc @achilleas-k

Reply:
I think it's okay to inject this as a file for now and make a dedicated stage out of it later.

Reply:
changed to append